chainer/function_node.py

```diff
@@ -1014,23 +1014,23 @@ def grad(outputs, inputs, grad_outputs=None, grad_inputs=None, set_grad=False,
     n_chx_inputs = sum([False if x is None else x._has_chainerx_array
                         for x in inputs])
     if n_chx_inputs == len(inputs):
-        if loss_scale is not None:
-            raise ValueError(
-                'loss_scale is not supported on chainerx.grad interface')
-
         # Need to access the arrays to invoke the chainer grad function
         if grad_outputs:
             grad_outputs_chx = [x._data[0] for x in grad_outputs]
         else:
             grad_outputs_chx = []
         outputs_chx = [x._data[0] for x in outputs]
         inputs_chx = [x._data[0] for x in inputs]
+        # pybind has issues when converting opt<int> -> opt<float>
+        if loss_scale:
```

Comment:
It seems to be evaluated to `False` when `loss_scale` is an integer `0`.
```suggestion
        if loss_scale is not None:
```